src/cmdc/datasets/official/CA/counties/sandiego.py

```diff
@@ -0,0 +1,85 @@
+import requests
+import pandas as pd
+
+from cmdc.datasets.official.base import ArcGIS
+
+
+class SanDiego(ArcGIS):
+    """
+    San Diego publishes their county level data in a dashboard that can
+    be found at:
+
+    https://www.sandiegocounty.gov/content/sdc/hhsa/programs/phs/community_epidemiology/dc/2019-nCoV/status.html
+
+    They don't seem to provide any simple interface to their data...
+    However, one can directly request data from arcgis itself which is
+    the route we take here. The San Diego arcgis service called
+    `CovidDashUpdate` had all relevant data within it, so we can only use
+    the one for now
+    """
+    ARCGIS_ID = "1vIhDJwtG5eNmiqX"
+    FIPS = 6073
+
+    def __init__(self):
+        # Default parameter values
+        params = {
+            "where": "0=0",
+            "outFields": "Date,Tests,Positives",
+            "returnGeometry": "false",
+            "f": "pjson"
+        }
+
+        super(SanDiego, self).__init__(params=params)
+
+        return None
+
+    def _insert_query(self, df, table_name, temp_name, pk):
+        out = f"""
+        INSERT INTO data.{table_name} (vintage, dt, fips, variable_id, value)
+        SELECT tt.vintage, tt.dt, tt.fips, mv.id as variable_id, tt.value
+        FROM {temp_name} tt
```

Comment:
This looks like the same exactly definition as the `CountyData` parent class

If it is we should delete this one

Reply:
Deleted.